### DISCUSSION TITLE
Sharper resolution for close twins

### DISCUSSION THREAD

**Discussion by RammyCa** [mtex-toolbox/mtex#1892] [Ask Anything]
Hi, I am investigating twinning using pole figures and my sample seems to have some really small twinning action going on, to the point that the ODF and sigma ODF dont really show much of a difference between these two isolated twins.

Is there any way to make the resolution better to catch details like this?

**Comment by lfgmorales** [comment]
Hi

You can try the approach of Daniel Savage for heavily twinned Ti

<https://www.sciencedirect.com/science/article/abs/pii/S1044580320322798>
sciencedirect.com<https://www.sciencedirect.com/science/article/abs/pii/S1044580320322798>
        [X] <https://www.sciencedirect.com/science/article/abs/pii/S1044580320322798>



Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 7 Jul 2023, at 16:16, RammyCa ***@***.***> wrote:

﻿

Hi, I am investigating twinning using pole figures and my sample seems to have some really small twinning action going on, to the point that the ODF and sigma ODF dont really show much of a difference between these two isolated twins.

Is there any way to make the resolution better to catch details like this?

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1892>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2C3YALOG5LN4Z562TXPAKZVANCNFSM6AAAAAA2B4B6HM>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1892", "url": "https://github.com/mtex-toolbox/mtex/discussions/1892", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
